tests/test_server.py

```diff
@@ -38,6 +38,12 @@
      ]
 
 
+@pytest.fixture(autouse=False)
```

Comment:
@PProfizi What's the default scope for that one ? Is it used then ?

Reply:
@anslpa Default scope is "each test", and since autouse=False and it does not appear anywhere, it will not be used.